Validation/SiOuterTrackerV/plugins/OuterTrackerMonitorTrackingParticles.cc

```diff
@@ -469,18 +473,18 @@ void OuterTrackerMonitorTrackingParticles::analyze(const edm::Event& iEvent, con
     res_eta->Fill(matchTrk_eta[j] - trkParts_eta[j]);
 
     // Eta and pT histograms for resolution
-    double pt_res = (matchTrk_pt[j] - trkParts_pt[j])/trkParts_pt[j];
-    double eta_res = matchTrk_eta[j] - trkParts_eta[j];
-    double phi_res = matchTrk_phi[j] - trkParts_phi[j];
-    double VtxZ_res = matchTrk_VtxZ[j] - trkParts_VtxZ[j];
-    double d0_res = matchTrk_d0[j] - trkParts_d0[j];
+    float pt_res = (matchTrk_pt[j] - trkParts_pt[j])/trkParts_pt[j];
+    float eta_res = matchTrk_eta[j] - trkParts_eta[j];
+    float phi_res = matchTrk_phi[j] - trkParts_phi[j];
+    float VtxZ_res = matchTrk_VtxZ[j] - trkParts_VtxZ[j];
+    float d0_res = matchTrk_d0[j] - trkParts_d0[j];
 
-    double tP_eta = trkParts_eta[j];
-    double tP_pt = trkParts_pt[j];
+    float tP_eta = trkParts_eta[j];
+    float tP_pt = trkParts_pt[j];
 
     // Fill resolution plots for different abs(eta) bins:
     // (0, 0.7), (0.7, 1.0), (1.0, 1.2), (1.2, 1.6), (1.6, 2.0), (2.0, 2.4)
-    if (abs(tP_eta) >= 0  && abs(tP_eta) < 0.7){
+    if (fabs(tP_eta) >= 0  && fabs(tP_eta) < 0.7){
```

Comment:
would not be better to add <cmath> and declare explicitly std::fabs ?

Reply:
@fabiocos, is that necessary? There seems to be plenty of instances in CMSSW where it is not explicitly declared. If you think it would be better, though, I will add the library.

Reply:
I would appreciate, replacing abs with std::abs is a cleaning that is suggested, see also the recommendations in the ORP spreadsheet https://docs.google.com/spreadsheets/d/1L_FVociAP-WcuSpt2vrM2R2E3R3hZ64JL47hItocT4o/edit#gid=975516649

Reply:
I will replace them, then! No problem